optuna/samplers/_tpe/sampler.py

```diff
@@ -94,16 +94,12 @@ def objective(trial):
         consider_prior:
             Enhance the stability of Parzen estimator by imposing a Gaussian prior when
             :obj:`True`. The prior is only effective if the sampling distribution is
-            either :class:`~optuna.distributions.UniformDistribution`,
```

Comment:
Same as optuna/multi_objective/samplers/_motpe.py. ☝️

Reply:
Done by keeping the changes from https://github.com/optuna/optuna/pull/3181!